scripts/check_frontend_coverage.py

```diff
@@ -267,6 +268,7 @@
     'ParamSpecsObjectFactory.ts',
     'ParamTypeObjectFactory.ts',
     'pencil-code-editor-rules.service.ts',
+    'pencil-code-editor-validation.service.ts',
```

Comment:
Why we haven't added all other spec filenames here which are in this PR?

Reply:
it is related to this PR.because I have added all the three filenames.